Mathlib/Data/Finset/Image.lean

```diff
@@ -79,6 +79,17 @@ theorem mem_map_equiv {f : α ≃ β} {b : β} : b ∈ s.map f.toEmbedding ↔ f
       simpa, fun h => ⟨_, h, by simp⟩⟩
 #align finset.mem_map_equiv Finset.mem_map_equiv
 
+/-- The `Finset` version of `Set.image_subset_iff`. -/
+theorem map_subset_iff {t : Finset β} {f : α ≃ β} : s.map f ⊆ t ↔ s ⊆ t.map f.symm := by
```

Comment:
That's incorrect. It's closer to something like `Equiv.subset_image'`. You will have rename existing lemmas if you want to fit this one in (and you get to fix the name of `Equiv.subset_image` while you're at it).

Reply:
Done.